fedimint-api-client/src/query.rs

```diff
@@ -177,7 +108,6 @@ pub struct ThresholdConsensus<R> {
 impl<R> ThresholdConsensus<R> {
     pub fn new(num_peers: NumPeers) -> Self {
         Self {
-            error_strategy: ErrorStrategy::new(num_peers.one_honest()),
```

Comment:
This renders `error_strategy` not needed, but if no existing (and forseeable) query strategy was making use of it, that's OK.